### PR TITLE
3 make base template and base css

### PR DESCRIPTION
**Pull request by Ozrei**

### CODE REVIEW

File: keyform/templates/keyform/home.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{% extends "keyform/base.html" %}
+	{% block content %}
+    Welcome to Keyform!            
+    {% endblock %}
```
Comment (enki211) [inline keyform/templates/keyform/home.html]: No newline at the eof

File: keyform/templates/keyform/base.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <title>{% block title %}Keyform{% endblock %}</title>
+    </head>
+    <body>
+        {% block body %}
+            {% block header %}
+                <div class = "title-bar">
+                    <div class = "title-bar-title">{% trans 'Keyform' %}</div>
+                </div>
+            {% endblock %}
+                {% block content %}
```
Comment (enki211) [inline keyform/templates/keyform/base.html]: Odd spacing

File: keyform/templates/keyform/base.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{% load static %}
+{% load i18n %}
+
+<!DOCTYPE html>
+<html>
+    <head>
+        {% block css %}
+            <link rel="stylesheet" type="text/css" href="{% static 'keyform/css/tablesorter_themes.css' %}"/>
+            <link rel="stylesheet" href="{% static 'keyform/css/foundation_css/foundation.css' %}"/>
```
Comment (enki211) [inline keyform/templates/keyform/base.html]: Maybe put a type="text/css"

File: keyform/templates/keyform/base.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            <link rel="stylesheet" href="{% static 'keyform/css/font-awesome-4.6.3/css/font-awesome.min.css' %}"/>
+            <link rel="stylesheet" href="{% static 'keyform/css/base.css' %}"/>
+        {% endblock %}
+        {% block javascript %}
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/jquery-2.1.4.min.js' %}"></script>
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/jquery.tablesorter.js' %}"></script>
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/jquery.tablesorter.widgets.js' %}"></script>
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/foundation.min.js' %}"></script>
+            <script type="text/javascript" src="{% static 'keyform/js/base.js' %}"></script>
```
Comment (tom08) [inline keyform/templates/keyform/base.html]: This file doesn't exist, there's no need to include it yet.

File: keyform/templates/keyform/base.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{% load static %}
+{% load i18n %}
+
+<!DOCTYPE html>
+<html>
+    <head>
+        {% block css %}
+            <link rel="stylesheet" type="text/css" href="{% static 'keyform/css/tablesorter_themes.css' %}"/>
```
Comment (tom08) [inline keyform/templates/keyform/base.html]: This file doesn't exist, there's no need to include it.

File: keyform/templates/keyform/base.html [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/jquery.tablesorter.widgets.js' %}"></script>
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/foundation.min.js' %}"></script>
+        {% endblock %}
+        <title>{% block title %}Keyform{% endblock %}</title>
+    </head>
+    <body>
+        {% block body %}
+            {% block header %}
+                <div class = "title-bar">
+                    <div class = "title-bar-title">{% trans 'Keyform' %}</div>
```
Comment (Patrick7321) [inline keyform/templates/keyform/base.html]: I realize that this is not my fault

File: keyform/templates/keyform/base.html [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+            <link rel="stylesheet" type="text/css" href="{% static 'keyform/css/font-awesome-4.6.3/css/font-awesome.min.css' %}"/>
+            <link rel="stylesheet" type="text/css" href="{% static 'keyform/css/base.css' %}"/>
+        {% endblock %}
+        {% block javascript %}
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/jquery-2.1.4.min.js' %}"></script>
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/jquery.tablesorter.js' %}"></script>
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/jquery.tablesorter.widgets.js' %}"></script>
+            <script type="text/javascript" src="{% static 'keyform/js/libraries/foundation.min.js' %}"></script>
+        {% endblock %}
+        <title>{% block title %}Keyform{% endblock %}</title>
```
Comment (Patrick7321) [inline keyform/templates/keyform/base.html]: Add a trans tag around the title

File: keyform/views.py [hunk 8]
```diff
@@ -2,5 +2,7 @@
 from __future__ import unicode_literals
 
 from django.shortcuts import render
+from django.views.generic.base import TemplateView
 
-# Create your views here.
+class HomeView(TemplateView):
+	template_name = "keyform/home.html"
```
Comment (enki211) [inline keyform/views.py]: No newline

**Pull request merged**